src/libraries/iKin/src/iKinFwd.cpp

```diff
@@ -1566,18 +1566,45 @@ iCubTorso::iCubTorso(const string &_type) : iKinLimb(string(""))
 /************************************************************************/
 void iCubTorso::allocate(const string &_type)
 {
+    double version;
+    size_t underscore = getType().find('_');
+    if (underscore != string::npos)
+    {
```

Comment:
The convention in this file is not to use curly brackets `{` for single operations.